packages/stripe-integration/src/stripev3/stripev3-payment-strategy.ts

```diff
@@ -202,12 +202,14 @@ export default class StripeV3PaymentStrategy implements PaymentStrategy {
 
             await this.paymentIntegrationService.submitPayment(paymentPayload);
         } catch (error) {
-            await this.processAdditionalAction(
-                this.handleEmptyPaymentIntentError(error, stripeError),
-                methodId,
-                shouldSaveInstrument,
-                shouldSetAsDefaultInstrument,
-            );
+            if (error instanceof Error || isRequestError(error)) {
```

Comment:
@PavlenkoM may I ask you to check this changes please?

Reply:
Changed `processAdditionalAction`'s signature in order to remove this type check.

Reply:
Should we use here `||`? I think `&&` will be better, otherwise why we need this type check?

Reply:
And also we are already had part of this check inside `processAdditionalAction` 
I think no need to double check `isRequestError`, can you please move this check inside `processAdditionalAction`
[https://github.com/bigcommerce/checkout-sdk-js/blob/master/packages/stripe-integration/src/stripev3/stripev3-payment-strategy.ts#L649](https://github.com/bigcommerce/checkout-sdk-js/blob/master/packages/stripe-integration/src/stripev3/stripev3-payment-strategy.ts#L649)